gt/main.c

```diff
@@ -2408,6 +2408,8 @@ uint32_t gt_cpu_to_be_32(uint32_t x);
 uint16_t gt_be_to_cpu_16(uint16_t x);
 uint32_t gt_be_to_cpu_32(uint32_t x);
 
+unsigned gt_lcore_id(void);
```

Comment:
I know that DPDK code uses `unsigned` to mean `unsigned int`, but we have been using `unsigned int` in our code. Could you replace `unsigned` with `unsigned int` in this patch?